tldw_Server_API/tests/DB_Management/test_media_db_schema_bootstrap.py

```
@@ -1987,3 +1988,606 @@ def test_ensure_media_schema_keeps_sqlite_schema_intact() -> None:
+        )
+        conn.commit()
+    finally:
+        conn.close()
+
+    db = MediaDatabase(str(db_path), client_id="schema-run-history-migration")
+    try:
+        db.close_connection()
+        db.backend.get_pool().close_all()
+        verification_db_path = tmp_path / "media_v23_verification.sqlite"
+        with sqlite3.connect(db_path) as source_conn:
+            with sqlite3.connect(verification_db_path) as dest_conn:
```

Ruff (0.15.9)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)